src/identities.sql

```diff
@@ -152,22 +152,37 @@ create trigger persons_channel_notify after insert or delete or update on person
 -- cannot drop this since default value of column depends on it
 -- so we always only replace it, unless all tables are dropped
 -- in which case we will recreate the default value anyways
-create or replace function generate_new_posix_id(table_name text, colum_name text)
+create or replace function generate_new_posix_id(
+    table_name text,
+    colum_name text,
+    objects_table text,
+    start_id int default 1000
+)
     returns int as $$
     declare current_max_id int;
     declare new_id int;
+    declare column_exists boolean;
```

Comment:
isn't `row_exists` a more accurate name?

Reply:
Come to think of it, maybe `posix_id_in_use` might be even more descriptive, the subsequent logic would read `if posix_id_in_use then ...`

Reply:
Agree on both points.